src/zarr/api/asynchronous.py

```diff
@@ -990,7 +992,7 @@ async def create(
         dtype = parse_dtype(dtype, zarr_format)
         if not filters:
             filters = _default_filters(dtype)
-        if not compressor:
```

Comment:
i haaaaaaate `if not x`! I'm guessing this was getting tripped because the empty tuple is falsey?

Reply:
No, this was correct before, the change is deliberate because the default value of `compressor` has changed.